deepmd/dpmodel/model/make_base_atomic_model.py

```diff
@@ -57,6 +57,42 @@ def get_nnei(self) -> int:
             """Returns the total number of selected neighboring atoms in the cut-off radius."""
             return self.get_nsel()
 
+        def get_dim_fparam(self) -> int:
+            """Get the number (dimension) of frame parameters of this DP."""
+            return 0
+
+        def get_dim_aparam(self) -> int:
+            """Get the number (dimension) of atomic parameters of this DP."""
+            return 0
+
+        def get_sel_type(self) -> List[int]:
+            """Get the selected atom types of this model.
+
+            Only atoms with selected atom types have atomic contribution
+            to the result of the model.
+            If returning an empty list, all atom types are selected.
+            """
+            return []
+
+        def get_numb_dos(self) -> int:
+            """Get the number of DOS."""
+            return 0
```

Comment:
number of dos should be the dim of output defined by output_def. we do not need the interface for querying it.